WalletWasabi/WabiSabi/Client/AmountDecomposer.cs

```diff
@@ -93,6 +79,12 @@ public IEnumerable<Output> Decompose(IEnumerable<Money> myInputCoinEffectiveValu
 		var smallestScriptType = Math.Min(ScriptType.P2WPKH.EstimateOutputVsize(), ScriptType.Taproot.EstimateOutputVsize());
 		var maxNumberOfOutputsAllowed = Math.Min(AvailableVsize / smallestScriptType, 10); // The absolute max possible with the smallest script type.
 
+		// If my input sum is smaller than the smallest denomination, then participation in a coinjoin makes no sense.
+		if (denoms.Min(x => x.EffectiveCost) > myInputSum)
```

Comment:
`CoinJoinCoinSelector.SelectCoinsForRound` could have a basic check similar to this, to abort CJ before the critical phase. Check could be like: `myInputSum > minRegistrableOutput`?

Reply:
Yup, it has. It's coming from `UtxoSelectionParameters`. Unit tests area also added for this.

Reply:
I do not find it.